mail_tracking/models/ir_mail_server.py

```diff
@@ -79,7 +79,7 @@ def send_email(self, message, mail_server_id=None, smtp_server=None,
                 smtp_server=smtp_server, smtp_port=smtp_port,
                 smtp_user=smtp_user, smtp_password=smtp_password,
                 smtp_encryption=smtp_encryption, smtp_debug=smtp_debug)
-        except Exception as e:
+        except Exception as e:  # pragma: no cover
             if tracking_email:
                 tracking_email.smtp_error(self, smtp_server_used, e)
             raise
```

Comment:
I have to raise the exception because i'm inheriting the method, so I have to propagate the result, even an exception